docs/book/src/reference/using_an_external_type.md

```diff
@@ -0,0 +1,155 @@
+# Using an External Type
+
+There are several different external types that may be referenced when writing a controller.
+* Custom Resource Definitions (CRD) that is defined in the current project `kubebuilder create api`.
+* Core Kubernetes Resources eg. Deployments or Pods.
+* CRD's that are created and installed in another project.
+* A Custom Resource defined via an API Aggregation (AA). Aggregated APIs are subordinate APIServers that sit behind the primary API server, which acts as a proxy.
+
+Currently Kubebuilder handles the first two, CRDs and Core Resources, seamlessly.  External CRDs and CRs created via Aggregation must be scaffolded manually.
+
+In order to use a Kubernetes Custom Resource that has been defined in another project
+you will need to have several items of information.
+* The Domain of the CR
+* The Group under the Domain 
+* The Go import path of the CR Type definition.
+
```

Comment:
Could we try to mainly add the content from https://book.kubebuilder.io/migration/manually_migration_guide_v2_v3.html#supportability as well?

Reply:
Content is gone, replaced the docs completely with a new version that is up to date